src/main/java/net/geforcemods/securitycraft/items/ItemSentryRemoteAccessTool.java

```diff
@@ -0,0 +1,175 @@
+			return ActionResult.newResult(EnumActionResult.PASS, stack);
+		}
+	}
+
+
+	@Override
+	public EnumActionResult onItemUse(EntityPlayer player, World world, BlockPos pos, EnumHand hand, EnumFacing facing, float hitX, float hitY, float hitZ){
+		ItemStack stack = player.getHeldItem(hand);
+
+		if(!world.isRemote){
+			//PlayerUtils.sendMessageToPlayer(player, "clicked", "true", TextFormatting.RED);
+
+			List<EntitySentry> sentries = world.getEntitiesWithinAABB(EntitySentry.class, new AxisAlignedBB(pos));
+			if(!sentries.isEmpty()) {
+				//PlayerUtils.sendMessageToPlayer(player, "is Sentry!!!", "true", TextFormatting.RED);
+			}
+
+
+			//if(BlockUtils.getBlock(world, pos) instanceof IExplosive){
+			if(!sentries.isEmpty()) {
+				//PlayerUtils.sendMessageToPlayer(player, "is Sentry?", "true", TextFormatting.RED);
+				BlockPos pos2 = sentries.get(0).getPosition();
+
+				if(!isSentryAdded(stack, world, pos2)){
+					int availSlot = getNextAvaliableSlot(stack);
+
+					if(availSlot == 0){
+						PlayerUtils.sendMessageToPlayer(player, ClientUtils.localize("item.securitycraft:remoteAccessSentry.name"), ClientUtils.localize("messages.securitycraft:srat.noSlots"), TextFormatting.RED);
+						return EnumActionResult.FAIL;
+					}
+
+					if(world.getTileEntity(pos2) instanceof IOwnable && !((IOwnable) world.getTileEntity(pos2)).getOwner().isOwner(player)){
+						PlayerUtils.sendMessageToPlayer(player, ClientUtils.localize("item.securitycraft:remoteAccessSentry.name"), ClientUtils.localize("messages.securitycraft:srat.cantBind"), TextFormatting.RED);
+						return EnumActionResult.FAIL;
+					}
+
+					if(stack.getTagCompound() == null)
+						stack.setTagCompound(new NBTTagCompound());
+
+					stack.getTagCompound().setIntArray(("sentry" + availSlot), new int[]{BlockUtils.fromPos(pos2)[0], BlockUtils.fromPos(pos2)[1], BlockUtils.fromPos(pos2)[2]});
+					SecurityCraft.network.sendTo(new PacketCUpdateNBTTag(stack), (EntityPlayerMP) player);
+					PlayerUtils.sendMessageToPlayer(player, ClientUtils.localize("item.securitycraft:remoteAccessSentry.name"), ClientUtils.localize("messages.securitycraft:srat.bound").replace("#", Utils.getFormattedCoordinates(pos2)), TextFormatting.GREEN);
+				}else{
+					removeTagFromItemAndUpdate(stack, pos2, player);
+					PlayerUtils.sendMessageToPlayer(player, ClientUtils.localize("item.securitycraft:remoteAccessSentry.name"), ClientUtils.localize("messages.securitycraft:srat.unbound").replace("#", Utils.getFormattedCoordinates(pos2)), TextFormatting.RED);
+				}
+			}
+			else
+				player.openGui(SecurityCraft.instance, GuiHandler.SRAT_MENU_ID, world, (int) player.posX, (int) player.posY, (int) player.posZ);
+		}
+
+		return EnumActionResult.SUCCESS;
+	}
+
+	
+	@Override
+	@SideOnly(Side.CLIENT)
+	public void addInformation(ItemStack par1ItemStack, World world, List<String> list, ITooltipFlag flag) {
```

Comment:
par1ItemStack -> stack

Reply:
Also transferred from MRAT item 
https://github.com/Geforce132/SecurityCraft/blob/1935d4b1079775b5815198dd536b86b88e1dab66/src/main/java/net/geforcemods/securitycraft/items/ItemMineRemoteAccessTool.java#L88

Reply:
Ok, I will fix those when the merge is done.